lua/starfall/libs_sh/net.lua

```diff
@@ -471,6 +471,31 @@ function net_library.readEntity(callback)
 		return instance.WrapObject(Entity(index))
 	end
 end
+	
```

Comment:
This shouldn't be added. Just use the entity functions. You can add a 'toNextbot' cast function similar to 'toHologram'